### DISCUSSION TITLE
OCSP Stapling Support

### DISCUSSION THREAD

**Discussion by mohitvaid** [syslog-ng/syslog-ng#4010] [Q&A] [answered]
As the latest version supports CRL via crl() tls option. 

Do we have any plan for the OCSP support? Any roadmap? 

PS: For TLS and MQTT TLS both.

@MrAnno

**Comment by MrAnno** [comment, marked as answer]
Please open an enhancement request about this, and we'll see what we can do.

Supporting OCSP stapling in the HTTP module is straightforward as libcurl just simply supports it.
For syslog network drivers, it requires a lot more work.
MQTT is based on paho.mqtt.c, so it needs some support from their side.

**Reply by mohitvaid** [reply]
Created https://github.com/syslog-ng/syslog-ng/issues/4013. Kindly check if this can be accommodated in the upcoming 3.7 release?
